app/services/protective-monitoring-service.js

```diff
@@ -0,0 +1,25 @@
+const { PublishEvent } = require('ffc-protective-monitoring')
+
+async function sendEvent (claimId, message) {
+  const protectiveMonitoring = new PublishEvent(process.env.PROTECTIVE_MONITORING_URL)
```

Comment:
This environment variable should be fed from the `config` values not direct from the environment.  That way we can validate it and clearer what the configuration for the application should be.